# GenBank Database Divisions

GenBank divisions are divided into two general categories and were described in an (Genome Research (1997) 7(10)) article by Ouellette and Boguski; the full-text article is available ([Database Divisions and Homology Search Files: A Guide for the Perplexed](/~/htgs/paper)). The "Organismal" category includes databases pertaining to sequences derived from specific organisms and the "Functional" databases pertain to different types of sequence data being collected. Sequence records exist only in one GenBank division. For example, the HTG division includes unfinished sequences (phases 0, 1, and 2) being generated from several different organisms. As a sequence is updated to phase 3, it is moved into the appropriate organismal division. For instance, human phase 3 (finished) HTG sequences are located in the PRI division. The GenBank divisions listed here represent the location of the annotated sequence records; for homology search purposes the records are reformatted and stored in the [BLAST databases](//www.ncbi.nlm.nih.gov/BLAST/Blast.cgi?CMD=Web&PAGETYPE=BLASTHome). The different database divisions currently available, as well as the related BLAST database, are listed below. An example of a submission (one accession number) that has progressed through phase 1, phase 2, and phase 3 is available ([Examples](/~/htgs/examples)).

**Organismal Divisions:**

<table border="0" cellpadding="4" cellspacing="0">

<tbody>

<tr>

<th align="left">Database</th>

<th align="left">Division</th>

<th align="left">BLAST</th>

<th align="left">Example</th>

</tr>

<tr>

<td align="left">BCT</td>

<td align="left">Bacterial sequences</td>

<td align="left">nr, month</td>

</tr>

<tr>

<td align="left">PRI</td>

<td align="left">Primate sequences</td>

<td align="left">nr, month</td>

<td>Human Phase 3</td>

</tr>

<tr>

<td align="left">ROD</td>

<td align="left">Rodent sequences</td>

<td align="left">nr, month</td>

</tr>

<tr>

<td align="left">MAM</td>

<td align="left">Other mammalian sequences</td>

<td align="left">nr, month</td>

</tr>

<tr>

<td align="left">VRT</td>

<td align="left">Other vertebrate sequences</td>

<td align="left">nr, month</td>

</tr>

<tr>

<td align="left">INV</td>

<td align="left">Invertebrate sequences</td>

<td align="left">nr, month</td>

<td align="left">Drosophila, C. elegans Phase 3</td>

</tr>

<tr>

<td align="left">PLN</td>

<td align="left">Plant and Fungal sequences</td>

<td align="left">nr, month</td>

<td align="left">Arabidopsis Phase 3</td>

</tr>

<tr>

<td align="left">VRL</td>

<td align="left">Viral sequences</td>

<td align="left">nr, month</td>

</tr>

<tr>

<td align="left">PHG</td>

<td align="left">Phage sequences</td>

<td align="left">nr, month</td>

</tr>

<tr>

<td align="left">RNA</td>

<td align="left">Structural RNA sequences</td>

<td align="left">nr, month</td>

</tr>

<tr>

<td align="left">SYN</td>

<td align="left">Synthetic and chimeric sequences</td>

<td align="left">nr, month</td>

</tr>

<tr>

<td align="left">UNA</td>

<td align="left">Unannotated sequences</td>

<td align="left">nr, month</td>

</tr>

</tbody>

</table>

**Functional Divisions:**

<table border="0" cellpadding="4" cellspacing="0">

<tbody>

<tr>

<th align="left">Database</th>

<th align="left">Division</th>

<th align="left">BLAST</th>

<th align="left">Example</th>

</tr>

<tr>

<td align="left">EST</td>

<td align="left">Expressed Sequence Tags</td>

<td align="left">dbest, month</td>

</tr>

<tr>

<td align="left">STS</td>

<td align="left">Sequence Tagged Sites</td>

<td align="left">dbsts, month</td>

</tr>

<tr>

<td align="left">GSS</td>

<td align="left">Genome Survey Sequences</td>

<td align="left">dbgss, month</td>

</tr>

<tr>

<td align="left">HTG</td>

<td align="left">High Throughput Genomic sequences</td>

<td align="left">htgs, month</td>

<td align="left">All Organisms: Phase 0, 1, and 2</td>

</tr>

</tbody>

</table>

**Phase 0** sequences are single-few pass reads of a single clone (not contigs usually).  
**Phase 1** sequences are unfinished, unordered, and contain gaps.  
**Phase 2** sequences are unfinished, ordered, and can contain one or more gaps.  
**Phase 3** sequences are high quality finished sequences that do not contain gaps.





